### DISCUSSION TITLE
cannot send iota out due to Password faulty / 24seedwords screendump unclear / pin correct / stronghold ok

### DISCUSSION THREAD

**Discussion by Aumy108** [iotaledger/firefly#8325] [Help & Support]
I cannot do anything with my iota on firefly due to password incorrect on both wallets (tried all passwords I normally use or could think of included what I noted down - no luck) my screendump of seed words seems to be incorrect as it does not like that. my restore from stronghold is ok and my pin is ok, so I can see all, but cannot get it out. tried to create a few new wallets through restore, but cannot change password due to incorrect seed words.. urgh.. Please help!

**Comment by bdavis1260** [comment]
Same here the passwords aren’t working even though they should. :/
